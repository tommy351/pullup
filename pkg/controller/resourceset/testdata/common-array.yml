---
apiVersion: pullup.dev/v1alpha1
kind: ResourceSet
metadata:
  name: test-46
  namespace: test
  uid: c37ee64f-5c69-4c53-8585-adc137d0acca
spec:
  number: 46
  resources:
    - apiVersion: apps/v1
      kind: Deployment
      metadata:
        name: test
      spec:
        template:
          spec:
            containers:
              - name: nginx
                args: ['a3', 'b3', 'c']
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test
  namespace: test
spec:
  selector:
    matchLabels:
      app: test
  template:
    metadata:
      labels:
        app: test
    spec:
      containers:
        - name: nginx
          image: nginx
          args: ['a1']
