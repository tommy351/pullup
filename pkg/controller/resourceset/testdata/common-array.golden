# This is a golden file. DO NOT EDIT.
- apiVersion: apps/v1
  kind: Deployment
  metadata:
    creationTimestamp: "1970-01-01T00:00:00Z"
    name: test-46
    namespace: test
    ownerReferences:
    - apiVersion: pullup.dev/v1alpha1
      blockOwnerDeletion: true
      controller: true
      kind: ResourceSet
      name: test-46
      uid: c37ee64f-5c69-4c53-8585-adc137d0acca
    resourceVersion: "0"
    uid: 00000000-0000-0000-0000-000000000000
  spec:
    selector:
      matchLabels:
        app: test
    strategy: {}
    template:
      metadata:
        labels:
          app: test
      spec:
        containers:
        - args:
          - a3
          - b3
          - c
          image: nginx
          name: nginx
          resources: {}
  status: {}
